packages/mdc-typography/README.md

```diff
@@ -139,7 +139,7 @@ $mdc-typography-styles-button: (
 
 Example: Overriding the global `font-family` property. 
 ```scss
-$mdc-typography-font-family: "Arial, Helvetica, sans-serif";
+$mdc-typography-font-family: unquote("Arial, Helvetica, sans-serif");
```

Comment:
This is probably not the only place we have this in sass...I'll check and possibly create an issue.

Reply:
I'm not finding any other usages like this. No issue.